lib/src/device.dart

```diff
@@ -1,20 +1,17 @@
 import 'dart:async';
-import 'dart:ui';
-
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 
-import 'extensions/vector2.dart';
-
-/// Some utilities that did not fit anywhere else.
+/// Class with utilities methods for controlling the device somehow (e.g. setting the screen as fullscreen)
```

Comment:
Provides methods for controlling the device (e.g. setting the screen to fullscreen).